Make "get help" more promiment on home page

AmberPoo1 opened the pull request:
Changed news-xmli.xml and _style.scss, add font weight css style to “get help”.
Resolved:  Make "get help" more prominent on home page #175

AmberPoo1 committed:
Changed news-xmli.xml and _style.scss, add font weight css style to
“get help”.
Resolved:  Make "get help" more prominent on home page #175

AmberPoo1 commented:
@alawvt, @amandafrench . Have add bold css style to "get help". Now the page shows like follows:
![screen shot 2016-02-07 at 7 05 00 pm](https://cloud.githubusercontent.com/assets/13734326/12876319/00a2c014-cdce-11e5-8389-8368c8271e6c.png)

alawvt commented:
The bold is visibly better in Safari and Chrome but not in Firefox, see screenshot below,
![a-boldfirefox](https://cloud.githubusercontent.com/assets/13037168/12926828/1f071c56-cf33-11e5-9171-98325a0d1866.png)
@AmberPoo1, would you see what you can do for Firefox? Thanks.

AmberPoo1 committed:
Change font weight of "get help"

In the pull request test, “font-weight: bold;” is not supported in
firefox browser.

alawvt commented:
@AmberPoo1, thanks, it looks good in Firefox, Safari, and Chrome.
